code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -16,9 +17,14 @@ Mineral Sheets
 
 GLOBAL_LIST_INIT(sandstone_recipes, list ( \
 	new/datum/stack_recipe("pile of dirt", /obj/machinery/portable_atmospherics/hydroponics/soil, 3, time = 1 SECONDS, max_per_turf = STACK_RECIPE_ONE_PER_TILE, on_floor = TRUE), \
+	new/datum/stack_recipe("sand", /obj/item/stack/sheet/mineral/sand, 1), \
 	new/datum/stack_recipe("sandstone door", /obj/structure/mineral_door/sandstone, 10, max_per_turf = STACK_RECIPE_ONE_PER_TILE, on_floor = TRUE), \
 	))
 
+GLOBAL_LIST_INIT(sand_recipes, list ( \
+	new/datum/stack_recipe("sandstone", /obj/item/stack/sheet/mineral/sandstone, 1), \
```

Comment:
I'm not a fan of this, potential infinite sandstone door spam is not okay.

Reply:
K i should probably make 1 sandstone worth 10 or 20 sand for ballance